test/observer.test.tsx

```diff
@@ -115,6 +115,52 @@ function runTestSuite(mode: "observer" | "useObserver") {
         })
     })
 
+    describe("double-rendering/StrictMode behavior", () => {
+        // tslint:disable: no-console
+        let originalConsoleError: typeof console.error
```

Comment:
Please use `jest-console` instead of this boilerplate :) It's even used in this test file already :)

Reply:
Yup. Will do. I don't normally use Jest, so I wasn't aware of that one.